Tests/SwiftProtobufPluginLibraryTests/Test_Descriptor.swift

```diff
@@ -26,7 +26,7 @@ extension Descriptor {
 final class Test_Descriptor: XCTestCase {
 
   func testParsing() throws {
-    let fileSet = try Google_Protobuf_FileDescriptorSet(serializedBytes: fileDescriptorSetBytes)
+    let fileSet = try Google_Protobuf_FileDescriptorSet(contiguousBytes: fileDescriptorSetBytes)
```

Comment:
I _think_ the tests can stay on `serializedBytes` now? (so this whole PR would be a lot smaller?)

Reply:
Yes, I held off from changing these and regenerating protos in case we didn't want to go for this approach, but I'll go ahead and make those changes now too